app/src/components/menu.ts

```diff
@@ -287,7 +321,7 @@ export function createMenu({
         {
           label: 'Quit',
           accelerator: 'Cmd+Q',
-          click: appQuit,
+          role: 'quit',
```

Comment:
For my info: what's the nuance between `click: appQuit` and `role: 'quit'` ?

Reply:
See above on Reload, but `role` passes the need for accelerators and functions to Electron to handle.

Reply:
Neat, didn't know. Thx 👍.